wrappers/csharp/cs-tutorial-5-load-json/ADP_TestLoadSettingsJson.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Intel.RealSense
+{
+    class ADP_TestLoadSettingsJson
+    {
+
+        public bool LoadSettingsJson(Device device)
+        {
+            var deviceName = device.Info[CameraInfo.Name];
+            var usbType = device.Info[CameraInfo.UsbTypeDescriptor];
+            string[] deviceNameParts = deviceName.Split(" ".ToCharArray());
+            var model = deviceNameParts.Last().Trim();
+
+
+            AdvancedDevice adv=null;
+            SerializableDevice ser = null;
+
+            Console.WriteLine($"Device model: '{model}'");
+            
+            var jsonFileName = $"ADP_{model}_TEST_JSON_USB{usbType}.json";
+            var jsonPath = System.IO.Path.Combine(System.IO.Directory.GetCurrentDirectory(), "JsonConfigs", jsonFileName);
+            Console.WriteLine($"Searching for a json test file: '{jsonFileName}'");
+            if (!File.Exists(jsonPath))
+            {
+                Console.WriteLine($"File not found, so no settings data will be loaded into a camera. Exiting...");
+                return false;
+            }
+
+            Console.WriteLine($"Json test file found: {jsonPath}. Loading it...");
+            if (model == "L515")
```

Comment:
`advanced_mode` is a derivation of `serializable_device`
I.e., we should be able to serialize json with just the latter...